cli/lib/build.js

```diff
@@ -143,27 +191,44 @@ const checkModuleValid = async (execPath)=> {
   }
 }
 
+const checkRebuildModules = async () => {
+  if (fs.existsSync(path.resolve(publicDir, 'static'))) {
```

Comment:
Reuse local public resource for new build.... I don't think it's a good idea.
e.g. member A fully built for commit A(change shell code) then publish to version, later member B fully built for commit B(change core code) and publish to version, finally member A partially built for commit C(change shell code), but depends on the built of commit A. Which would drop commit B feature.
Another problem is if I don't release project for more than three days, then I don't know whether my local resource is still valid, so I have to make full building again.
Anyway, this pr can merge for now. I'll make adjustment later.